AirCasting/CreateSessionViews/ChooseCustomLocationView.swift

```diff
@@ -89,4 +95,17 @@ struct ChooseCustomLocationView: View {
             }
         )
     }
+    
+    private func updateTrackerWithNewLocation(_ loc: CLLocationCoordinate2D?) {
+        guard let loc else { return }
+        locationTracker.locationSource = loc
+        ApplocationTracker.location.value = .init(latitude: loc.latitude,
```

Comment:
Are you sure we want to update this AppLocationTracker location here? Shouldn't it always have actual location of the device? Why do we need to change it here?